README.md

```diff
@@ -1,3 +1,19 @@
 # PureScript-Cookbook
 
 An unofficial Cookbook for PureScript
+
+The recipes folder contains all the recipes in this repo in no particular order.
+
+## Contributing new recipes
+
+Follow these instructions for contributing new recipes:
+1. Pick an existing recipe to duplicate as a starting point. The `Template` recipe is probably the simplest".
+1. Rename the copied folder to a short description of the problem you're trying to solve using PascalCase (e.g. RoutingPushHalogen)
+1. If your recipe is incompatible with the browser enviornment (e.g. reading local files with node), delete the `dev` directory. Note that just logging to the console **is** supported in the browser.
+1. Use your recipe name to prefix all modules. Do a quick search to find all instances of the old recipe name to replace (e.g. `grep -r RoutingPushHalogen .`).
+1. If needed, update the `packages.dhall` file to include more packages.
```

Comment:
While we'll need to allow this, we need to be careful about attack vectors, too. So, not necessarily here, but somewhere else (e.g. `CONTRIBUTING.md`), we should specify that one should add their packages to the package set. The attack vector isn't eliminated by doing that, but it'll be exposed to more eyes I think...?

Reply:
I think we could also just be very strict and insist on latest package set only.

Reply:
I've opened #8 to track this.